eras/mary/impl/cddl-files/mary.cddl

```diff
@@ -118,26 +127,42 @@ ipv6 = bytes .size 16
 
 major_protocol_version = 1 .. 3
 
+metadata = {* transaction_metadatum_label => transaction_metadatum}
+
 metadata_hash = $hash32
 
 mint = multiasset<int64>
 
-move_instantaneous_reward = [0 / 1, {* stake_credential => coin}]
+; The first field determines where the funds are drawn from.
+;   0 denotes the reserves,
+;   1 denotes the treasury.
+; If the second field is a map, funds are moved to stake credentials.
+; Otherwise, the funds are given to the other accounting pot.
+; NOTE:
+;   This has been safely backported to Shelley from Alonzo.
```

Comment:
this comment does not seem to belong here

Reply:
yeah, but I guess it's not easy because the type is shared between the eras. Maybe you can just change it to "This has been safely backported from Alonzo" without the reference to Shelley in a future PR.